core/src/main/java/com/yahoo/ycsb/generator/FacebookValueGenerator.java

```diff
@@ -0,0 +1,91 @@
+package com.yahoo.ycsb.generator;
+
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.PrintStream;
+
+import com.yahoo.ycsb.Utils;
+
+class GeneralizedParetoGenerator extends IntegerGenerator{
+	private static final double loc = 15; // location
+	private static final double scale = 214.476; // scale
+	private static final double shape = 0.348238; //shape
+	
+	@Override
+	public int nextInt() {
+		double u = Utils.random().nextDouble();
+		double ret = loc + scale *(Math.pow(u, -shape) -1)/shape;
+		return (int)ret;
+	}
+
+	@Override
+	public double mean() {
+		// TODO Auto-generated method stub
+		return 0;
+	}
+	
+}
+
```

Comment:
need javadocs. I don't see a description of this in the top level. is it also from the paper?

Please find a suitable name that does not use Facebook's trademarks.